packages/front-end/components/SavedGroups/ConditionGroups.tsx

```diff
@@ -198,7 +227,12 @@ export default function ConditionGroups({ groups, mutate }: Props) {
                   {!items.length && isFiltered && (
                     <tr>
                       <td
-                        colSpan={canUpdate || canDelete ? 6 : 5}
+                        colSpan={
+                          canUpdate({ projects: [project] }) ||
+                          canDelete({ projects: [project] })
+                            ? 6
+                            : 5
```

Comment:
Are these column counts accurate still?  It looks like you added 1 additional column to the table, so will this have to go to 6/7 instead of 5/6?